src/index.html

```diff
@@ -15,18 +15,38 @@
       rel="stylesheet"
       href="style.css"
     />
+    <link
+      rel="stylesheet"
+      href="fonts/Avenir-Book.ttf"
+    />
+    <link
+      rel="stylesheet"
+      href="fonts/Avenir-Heavy.ttf"
+    />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="container"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
Lines of code should have 80 characters max. The `placeholder` value here is too long and should be split into multiple lines with proper indentation.